operator/src/test/java/io/strimzi/kafka/access/SecretDependentResourceTest.java

```diff
@@ -85,15 +86,24 @@ void testSecretDataWithTls() {
         final String cert = encodeUtf8("-----BEGIN CERTIFICATE-----\nMIIFLTCCAx\n-----END CERTIFICATE-----\n");
         final Map<String, String> certSecretData = new HashMap<>();
         certSecretData.put("ca.crt", cert);
-        final Secret certSecret = ResourceProvider.getStrimziSecret(KafkaResources.clusterCaCertificateSecretName(KAFKA_NAME), KAFKA_NAMESPACE, KAFKA_NAME);
-        certSecret.setData(certSecretData);
+//        final Secret certSecret = ResourceProvider.getStrimziSecret(KafkaResources.clusterCaCertificateSecretName(KAFKA_NAME), KAFKA_NAMESPACE, KAFKA_NAME);
+//        certSecret.setData(certSecretData);
```

Comment:
Should it still be here or should it be removed?

Reply:
See below.